services/libs/integrations/src/integrations/premium/hubspot/field-mapper/organizationFieldMapper.ts

```diff
@@ -269,13 +234,16 @@ export class HubspotOrganizationFieldMapper extends HubspotFieldMapper {
       )
     }
 
+    // TODO uros check if this is verified or not with anil
```

Comment:
we can assume it's verified if it's coming from hubspot i guess

Reply:
It actually doesn't matter - I checked with Joan and LF is not gonna be using HubSpot and it's probably gonna get removed (the code for integration) so at this point we don't have to touch it.